### DISCUSSION TITLE
supabase edge function capacity

### DISCUSSION THREAD

**Discussion by PierreTTT** [supabase/supabase#28722] [Questions]
If i have a feature that requires the edge function to run for around 10-20 sec when a user use it. What is the concurrent limitation of it? can it support in case that there is a spike in the traffic of 1000 to 10 000 users?

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/functions/limits
That has the limits.   It is going depend on what you are doing for that 10 to 20 seconds.  If a lot of CPU work then you likely have an issue. 
Edge function calls should kick off new edge functions as needed.
